src/App.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+} from 'react-router-dom';
+import Navbar from './components/navbar';
+import Books from './components/books';
+import Form from './components/form';
+import Categories from './components/categories';
+
+const App = () => (
+  <Router>
+    <div>
+      <Navbar />
+      <Routes>
+        <Route exact path="/books" element={<Books />} />
+        <Route exact path="/" element={<Books />} />
```

Comment:
Line 17 is not needed because of line 18

Reply:
Thanks

Reply:
Thanks. Solved.